apps/meteor/client/components/MarkdownText.tsx

```diff
@@ -110,7 +110,7 @@ const MarkdownText: FC<Partial<MarkdownTextParams>> = ({
 		})();
 
 		return preserveHtml ? html : html && sanitizer(html, { ADD_ATTR: ['target'] });
-	}, [content, preserveHtml, sanitizer, markedOptions, parseEmoji]);
+	}, [content, preserveHtml, sanitizer, markedOptions, parseEmoji, variant]);
```

Comment:
🤔

Reply:
typescript complained that it was being used without being depended on

Reply:
it does affect the CI, breaking on eslint[react-hooks/exhaustive-deps](https://github.com/facebook/react/issues/14920)

Reply:
No but, can you try updating with latest of develop? I'm curious why this is not failing on any other place 👀

Reply:
done